geppetto.js/geppetto-ui/src/dicom-viewer/DicomViewer.js

```diff
@@ -813,7 +813,16 @@ class DicomViewer extends Component {
   }
 }
 
-DicomViewer.defaultProps = { onLoaded: () => {}, };
+DicomViewer.defaultProps = { 
+  onLoaded: () => {},
+  mode: 'coronal',
+  orientation: 'goToPoint',
+  onClick: 'goToPoint',
+  onCtrlClick: 'goToPoint',
+  onShiftClick: 'goToPoint',
+  onDoubleClick: 'goToPoint',
+  showDownloadButton: false,
+};
 
 
 DicomViewer.propTypes = {
```

Comment:
I believe onClick, onCtrlClick, onShiftClick, onDoubleClick shouldn't be `any` but rather a string (one of  'goToPoint', 'goToSingleView', 'togglMode'), or a function.

Reply:
I also changed 'togglMode' to 'toggleMode'. I made the example DicomViewer match this too.

This is how the DicomViewer Props look like with the [new format](https://github.com/MetaCell/geppetto-meta/blob/1737735290f9fb5379d8c560ee57203ef3033de2/geppetto.js/geppetto-ui/src/dicom-viewer/DicomViewer.js#L848):
![image](https://user-images.githubusercontent.com/40673427/153307013-2670072f-6cd0-4dc5-ab78-d76294502eb5.png)

Let me know if this is the right way.

Reply:
Regarding the change from togglMode to toggleMode I think that's all right. I pushed a [new commit](https://github.com/MetaCell/geppetto-meta/commit/e770b7eb9aeaf0305a8666ced87d3862b3cc6c1f) to fix one thing you forgot to change. The idea of those 'modes' is to allow the user to choose some default onX strategies. That works because we have those strategies implemented in the some methods inside the DicomViewer component. When the user chooses 'toggleMode' we map it to the method toggleMode. Because this is using eval (which is something that we probably should change as it is not very safe/pretty) the 'mode' needs to match with the name of the method.

New format looks very nice in paper. I don't like how it looks in the showcase but I think we can address that in #285.